build-configs/aschaffenburg/index.js

```diff
@@ -62,9 +62,8 @@ export const webAschaffenburgBuildConfig: WebBuildConfigType = {
   ...commonAschaffenburgBuildConfig,
   mainImprint,
   itunesAppId: APPLE_APP_ID,
-  manifestUrl: '/manifest.json',
   icons: {
-    appLogo: '/app-logo.png',
+    appLogo: '/app-logo.svg',
```

Comment:
Are we sure an svg works here? not sure where this is used.

Reply:
hmm changed it to png. Just in case. But I think this is only used in web an should not cause many problems, as the other assets also contain svgs